cmd/ci-secret-bootstrap/main.go

```diff
@@ -167,10 +182,17 @@ func (o *options) validateCompletedOptions() error {
 	return nil
 }
 
+type attributeType string
+
+const (
+	attributeTypePassword = "password"
```

Comment:
This is a `string`, not an `attributeType`

Reply:
Updated.